tsl/src/continuous_aggs/create.c

```diff
@@ -1835,6 +1835,29 @@ tsl_process_continuous_agg_viewstmt(Node *node, const char *query_string, void *
 	timebucket_exprinfo = cagg_validate_query((Query *) stmt->into->viewQuery);
 	cagg_create(stmt, &viewstmt, (Query *) stmt->query, &timebucket_exprinfo, with_clause_options);
 
+	/* Refreshing unless requested to not refresh. Cannot use
+	 * DirectFunctionCall since we pass NULL parameters to function. */
+	if (!stmt->into->skipData)
```

Comment:
Postgres has the following check for materialized view creation:

```
	/* Check that conflicting options have not been specified. */
	if (concurrent && stmt->skipData)
		ereport(ERROR,
				(errcode(ERRCODE_SYNTAX_ERROR),
				 errmsg("CONCURRENTLY and WITH NO DATA options cannot be used together")));
```

Do we want to add the same or possibly block `CONCURRENTLY` altogether? Could also be done separate from this PR.

Reply:
I think we should stick to the same semantics as PostgreSQL as far as possible, but this check is for `REFRESH MATERIALIZED VIEW` and not for `CREATE MATERIALIZED VIEW`, right? This PR does not touch `REFRESH MATERIALIZED VIEW`.

Reply:
Yes, I think you are right. That's for refresh.